### ISSUE TITLE
React documentation seems to be out of date

### ISSUE THREAD

**Issue by nsantacruz** [UniversalDataTool/universal-data-tool#345]
I tried to follow the documentation on this [page](https://github.com/UniversalDataTool/universal-data-tool/wiki/Usage-with-React) but I got an error from webpack saying that `UniversalDataViewer` does not exist.

I checked in `universal-data-tool/components` and could not find the component. The closest component I found was `UniversalSampleEditor` which takes one sample and allows you to edit it. This solved my use-case.

Anyway, the issue is that the documentation doesn't seem to be correct. Happy to make a PR if it's clear what the documentation should say.

**Comment by seveibar** [comment]
Yep! The naming changed in 0.14 to UniversalSampleEditor, the docs are OK for 0.13.x

PR appreciated!

**Comment by seveibar** [comment]
There were also some minor prop changes IIRC
